da/celestia/celestia.go

```diff
@@ -328,7 +321,14 @@ func (c *DataAvailabilityLayerClient) retrieveBatches(daMetaData *da.DASubmitMet
 	var batch pb.Batch
 	err = proto.Unmarshal(blob.Data, &batch)
 	if err != nil {
-		c.logger.Error("Unmarshal block.", "daHeight", daMetaData.Height, "error", err)
+		c.logger.Error("Unmarshal blob.", "daHeight", daMetaData.Height, "error", err)
+		return da.ResultRetrieveBatch{
+			BaseResult: da.BaseResult{
+				Code:    da.StatusError,
+				Message: err.Error(),
+				Error:   da.ErrBlobNotParsed,
+			},
+		}
```

Comment:
I now see that we write to celestia a batch which has:
1. LastBatch
2. DRSVersion 

but why is that info needs to be on celestia? 

also not sure it's backward compatible with unmrashaling old batch structures.